Analytics/Classes/Integrations/SEGAliasPayload.m

```diff
@@ -13,4 +25,17 @@ - (instancetype)initWithNewId:(NSString *)newId
     return self;
 }
 
+- (instancetype)initWithBuilder:(SEGAliasPayloadBuilder *)builder
+{
+    if (self = [super initWithBuilder:builder]) {
+        NSParameterAssert(builder);
+
+        NSString *theNewId = [builder.theNewId copy];
```

Comment:
Try taking it out :)

It's an objective c thing. You can't use new as a prefix 
http://stackoverflow.com/questions/24308162/property-name-starting-with-new-prefix-leads-to-bad-access-error-in-ios